Gemfile

```diff
@@ -4,6 +4,7 @@ gem 'rake'
 gem 'mumukit', github: 'mumuki/mumukit', tag: 'v0.4.0'
 gem 'mumukit-inspection', github: 'uqbar-project/mumukit-inspection', branch: 'master'
 gem 'stones-spec', github: 'uqbar-project/stones-spec', branch: 'master'
+gem 'recursive-open-struct', github: 'mumuki/recursive-open-struct', branch: 'feature-use-symbols-for-hash-representation'
```

Comment:
@flbulgarelli, before you complain I tell you that [I've already sent a PR](https://github.com/aetherknight/recursive-open-struct/pull/34). The thing is that I don't know if it will be accepted because it's a breaking change that perhaps the author doesn't want, we'll see.